libraries/AP_HAL_ChibiOS/hwdef/MFE_POS3_CAN/hwdef.dat

```diff
@@ -0,0 +1,108 @@
+# hw definition file for processing by chibios_pins.py
+# MCU class and specific type
+
+#AUTOBUILD_TARGETS None
+
+# MCU class and specific type
+MCU STM32F4xx STM32F427xx
+
+FLASH_RESERVE_START_KB 36
+
+STORAGE_FLASH_PAGE 16
+define HAL_STORAGE_SIZE 8192
+
+# board ID for firmware load
+APJ_BOARD_ID 6101
```

Comment:
You need to reserve this board ID in `Tools/AP_Bootloader/board_types.txt` and then use the name you use in that file in here.